app/models/apple/episode.rb

```diff
@@ -9,33 +9,23 @@ class Episode
     def self.get_episodes(api, episodes)
       return if episodes.empty?
 
-      resp =
-        api.bridge_remote("getEpisodes", episodes.map(&:get_episode_bridge_params))
-
-      # TODO: handle errors
-      episode_bridge_results = api.unwrap_response(resp)
-      episode_bridge_results
+      api.bridge_remote_and_retry!("getEpisodes", episodes.map(&:get_episode_bridge_params))
     end
 
     def self.create_episodes(api, episodes)
       return if episodes.empty?
 
-      resp =
-        api.bridge_remote("createEpisodes", episodes.map(&:create_episode_bridge_params))
+      episode_bridge_results = api.bridge_remote_and_retry!("createEpisodes",
```

Comment:
Looks like these calls could now throw errors (if the remote call fails enough retries).

Will these eventually bubble up to a worker/job then?  Or does this create `apple_podcast_deliveries` rows for failures?

/me reads on!

Reply:
Makes good sense.  I'd rather see the errors, when initially doing this all manually.